src/components/search/stats.tsx

```diff
@@ -13,7 +13,7 @@ const CustomStats: React.FunctionComponent<CustomStatsProps> = ({
   return !searchQuery || /^\s*$/.test(searchQuery) ? (
     <div />
   ) : (
-    <div className="sm:text-lg flex items-baseline flex-nowrap overflow-hidden max-w-full flex-grow pt-5 px-3">
+    <div className="sm:text-lg flex items-baseline flex-nowrap overflow-hidden max-w-full  pt-5 px-3">
```

Comment:
Having flex-grow on the stats component created awkard space at the top of the search results. Moved this to the paginating component and it looks better to me.

reference: 
![Resources for Web Developers in 2022  egghead io](https://user-images.githubusercontent.com/6188161/167166682-dce3f2f7-13f8-4c02-a83f-bf6ae36bd34e.png)